src/spdx2/agent/spdx2.php

```diff
@@ -436,8 +436,12 @@ protected function addScannerResults(&$filesWithLicenses, ItemTreeBounds $itemTr
     {
       $reportedLicenseId = $this->licenseMap->getProjectedId($row['rf_fk']);
       $shortName = $this->licenseMap->getProjectedShortname($reportedLicenseId);
-      if ($shortName != 'No_license_found' && $shortName != 'Void') {
-        $filesWithLicenses[$row['uploadtree_pk']]['scanner'][] = $shortName;
+      if ($shortName != 'Void') {
+        if($shortName != 'No_license_found'){
+          $filesWithLicenses[$row['uploadtree_pk']]['scanner'][] = $shortName;
+        }else{
+          $filesWithLicenses[$row['uploadtree_pk']]['scanner'][] = "";
```

Comment:
I like the `NOASSERTION` more because it represents the same rationale as with FOSSology and its "no license known": actually there might be a license, but the scanner did not find it. As with `NONE` it sounds like there is really nothing there.